libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -218,6 +243,374 @@ function BallotPageFrame({
+  election,
+  contest,
+}: {
+  election: Election;
+  contest: CandidateContestStruct;
+}) {
+  const voteForText = {
+    1: hmpbStrings.hmpbVoteFor1,
+    2: hmpbStrings.hmpbVoteFor2,
+    3: hmpbStrings.hmpbVoteFor3,
+    4: hmpbStrings.hmpbVoteFor4,
+    5: hmpbStrings.hmpbVoteFor5,
+    6: hmpbStrings.hmpbVoteFor6,
+    7: hmpbStrings.hmpbVoteFor7,
+    8: hmpbStrings.hmpbVoteFor8,
+    9: hmpbStrings.hmpbVoteFor9,
+    10: hmpbStrings.hmpbVoteFor10,
+  }[contest.seats];
+  if (!voteForText) {
+    throw new Error(
+      `Unsupported number of seats for contest: ${contest.seats}`
+    );
+  }
+
+  return (
+    <Box
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        padding: 0,
+      }}
+    >
+      <ContestHeader>
+        <DualLanguageText delimiter="/">
+          <h3>{electionStrings.contestTitle(contest)}</h3>
+        </DualLanguageText>
+        <DualLanguageText delimiter="/">
+          <div>{voteForText}</div>
+        </DualLanguageText>
+        {contest.termDescription && (
+          <DualLanguageText delimiter="/">
+            <div>{electionStrings.contestTerm(contest)}</div>
+          </DualLanguageText>
+        )}
+      </ContestHeader>
+      <ul>
+        {contest.candidates.map((candidate, i) => {
+          const partyText =
+            election.type === 'primary' ? undefined : (
+              <CandidatePartyList
+                candidate={candidate}
+                electionParties={election.parties}
+              />
+            );
+          const optionInfo: OptionInfo = {
+            type: 'option',
+            contestId: contest.id,
+            optionId: candidate.id,
+          };
+          return (
+            <li
+              key={candidate.id}
+              style={{
+                padding: '0.375rem 0.5rem',
+                borderTop:
+                  i !== 0 ? `1px solid ${Colors.DARK_GRAY}` : undefined,
+              }}
+            >
+              <div
+                style={{
+                  display: 'flex',
+                  gap: '0.5rem',
+                }}
+              >
+                <div style={{ flex: 1, textAlign: 'right' }}>
+                  <strong>{candidate.name}</strong>
+                  {partyText && (
+                    <DualLanguageText delimiter="/">
+                      <div>{partyText}</div>
+                    </DualLanguageText>
+                  )}
+                </div>
+                <BubbleWrapper optionInfo={optionInfo} />
+              </div>
+            </li>
+          );
+        })}
+        {contest.allowWriteIns &&
+          range(0, contest.seats).map((writeInIndex) => {
+            const optionInfo: OptionInfo = {
+              type: 'write-in',
+              contestId: contest.id,
+              writeInIndex,
+              writeInArea: {
+                top: 0.8,
+                right: -0.8,
+                bottom: 0.3,
+                left: 8.8,
+              },
```

Comment:
`writeInArea` has been updated

Reply:
The write-in area looks a little bit closer to the write-in line than the top of the box. That can sometimes be an issue when the ballot is skewed (as the line can overlap with the box). I might move it up a tiny bit. Though I'd be curious how it compares to the VX default ballot template - probably they should match.